lib/activity/router.js

```diff
@@ -55,6 +58,9 @@ module.exports = ({ models }) => {
             return;
           }
 
+          // Delay GitHub API calls to avoid replication lag
+          context.github.hook.before('request', avoidReplicationLag());
+
```

Comment:
Any way to remove this duplication? Maybe switch the conditional above to be `if (!subscription.creatorId)`, do the permission check, and then only call `callback` once at the end?

Reply:
I like that 👍